terraform/templates/lambda_invoke_policy.json

```diff
@@ -0,0 +1,12 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "lambda:InvokeFunction"
+            ],
+            "Resource": "*"
```

Comment:
Better to limit Resources to the one lambda function you want to invoke. Please only list the `ARN` of the lambda function here.